giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/users/UsersManager.java

```diff
@@ -123,6 +145,9 @@ public void approveUserAccount(String username) {
         if (!user.getRoles().contains(GilesRole.ROLE_USER.name())) {
             user.getRoles().add(GilesRole.ROLE_USER.name());
         }
+        if(user.getRoles().contains(GilesRole.ROLE_ADMIN.name())) {
```

Comment:
this should never be the case. when a user is first approved, it should not have the admin role yet.